test/quick-test.ts

```diff
@@ -3,21 +3,30 @@ import { starknet } from "hardhat";
 import { StarknetContract, StarknetContractFactory } from "hardhat/types/runtime";
 import { TIMEOUT } from "./constants";
 
-describe("Starknet", function() {
+async function quickTest(artifactsPathSegment: string) {
```

Comment:
Instead of modifying the existing quick-test.ts file, I'd rather you created a new one. In fact, if you created a new one, you wouldn't have to modify contract-factory-test.ts or path-test.ts - you could test both the loading and using of the loaded contract factory in the same new test file.

Reply:
Created new test